### PR TITLE
provider/openstack: Fix crash when access_network was not defined

### LINKED ISSUE #4954: Openstack Provider crash in 0.6.11
[WINDOWS]

After upgrading to 0.6.11 my terraforms just crash, if i'm rolling back to 0.6.10 it works again. 

[crash.txt](https://github.com/hashicorp/terraform/files/114200/crash.txt)

### PR DESCRIPTION
**Pull request by jtopjian**
Fixes #4954

### CODE REVIEW

File: builtin/providers/openstack/resource_openstack_compute_instance_v2.go [hunk 1]
```diff
@@ -998,7 +998,7 @@ func getInstanceAccessAddresses(d *schema.ResourceData, networks []map[string]in
 			hostv6 = n["fixed_ip_v6"].(string)
 		}
 
-		if n["access_network"].(bool) {
+		if ok, an := n["access_network"].(bool); ok && an {
```
Comment (jen20) [inline builtin/providers/openstack/resource_openstack_compute_instance_v2.go]: Should these be named the opposite way around for clarity? Appreciate that it doesn't actually matter here, but the "ok" is generally the second parameter of the type assertion?

**Comment by jen20** [comment]
Hi @jtopjian! I've left a comment in line - the nature of the fix looks fine, with one minor nitpick. Thanks for opening the PR!

File: builtin/providers/openstack/resource_openstack_compute_instance_v2.go (hunk 1 above)
Comment (jtopjian) [inline builtin/providers/openstack/resource_openstack_compute_instance_v2.go]: Yes indeed. I should really just step away from any keyboard for the rest of the day.

**Comment by jtopjian** [comment]
Thanks for the review. Should be good now.

**Comment by jtopjian** [comment]
@phinze I was swamped at work and wanted to fix this bug ASAP. Now that the dust has settled, I've had some time to review and I'm utterly confused as to why this bug was triggered. Most specifically, I was only able to recreate this bug by compiling Terraform to a binary and then applying a fresh config. This bug didn't trigger during any acceptance tests nor on existing configurations when used as a binary. Any ideas?

**Comment by krjensen** [comment]
Can we soon get a 0.6.12 including this patch? :)

**Pull request merged**